backends/platform/sdl/sdl-window.cpp

```diff
@@ -299,6 +303,9 @@ float SdlWindow::getDpiScalingFactor() const {
 }
 
 float SdlWindow::getSdlDpiScalingFactor() const {
+	if (ConfMan.hasKey("forced_dpi_scaling"))
+		return ConfMan.getInt("forced_dpi_scaling") / 100.f;
+
```

Comment:
I don't think adding it here is correct. This function is used to convert cursor position between the drawable area and the SDL window. Thus using the window size and drawable size to get that factor should always be correct.